registry/mdns_registry.go

```diff
@@ -138,18 +149,19 @@ func newRegistry(opts ...Option) Registry {
 	}
 
 	// set the domain
-	domain := mdnsDomain
+	defaultTLD := defaultTLD
```

Comment:
this seems wrong?

Reply:
Why so?

Reply:
I'll rename TLD => Domain to avoid confusion.